.gitlab/ci/.gitlab-ci.slack-notify.yml

```diff
@@ -1,11 +1,11 @@
 
 default:
-  image: docker-io.art.code.pan.run/devdemisto/gitlab-content-ci:1.0.0.49555
```

Comment:
It can technically, `docker-io.art.code.pan.run` is a simple `docker.io` caching proxy, we can split the rest to varaibles, but the purpose of this PR is to make this repo dynamic enough to be easily transferred between GitLab instances